src/main.c

```diff
@@ -20,21 +20,62 @@ static char const * io500_phase_str[IO500_SCORE_LAST] = {
   "MD",
   "BW"};
 
-static void init_dirs(void){
+static void prepare_aiori(void){
   // check selected API, might be followed by API options
   char * api = strdup(opt.api);
-  char * sep = strstr(api, " ");
-  if(sep){
-    *sep = '\0';
+  char * token = strstr(api, " ");
+
+  option_help options [] = {
+    LAST_OPTION
+  };
+  options_all_t * global_options = airoi_create_all_module_options(options);
+  if(token){
+    *token = '\0';
     opt.apiArgs = strdup(opt.api);
     opt.api = api;
+
+    // parse the API options, a bit cumbersome at the moment
+    // find the next token for all the APIs
+    token++;
+    for(char * p = token ; ; p++){
+      if(*p == ' '){
+        *p = '\0';
+        if( p - token > 1 && *token != ' '){
```

Comment:
Does this work for multi-byte unicode characters? I am thinking of the east asian machines potentially having native language directory names using unicode multi-byte characters.

Reply:
I don't think so, the configuration file must be written in ASCII